sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+friendly_name: Newtab Daily Aggregations
+description: |-
+  A daily aggregation of newtab metrics.
```

Comment:
Would it be possible to derive this data from `newtab_visits_daily_v2`? If not, what is missing?

Reply:
> Would it be possible to derive this data from `newtab_visits_daily_v2`? If not, what is missing?

@lucia-vargas-a this is indeed a very good suggestion! I feel doing this will give us better accuracy than an building a client level intermediate table.

This brings us to a follow-up question:
If we have a visit level table and use that to compute the final aggregations (rollup to get counts of client_ids and visit_ids), do we then need a client level table? 
Do we have any specific use cases for the client level table?

cc: @florezita @jaredsnyder

Reply:
Table at the client and day level is useful for experiments.  Experiment metrics are calculated at the client+day level for the most part

Reply:
yes-- client-day level tables support experiment outcomes in jetstream, for example, as defined [here](https://github.com/mozilla/metric-hub/blob/main/definitions/firefox_desktop.toml#L1020) in metric-hub